src/main/java/com/google/sps/servlets/PostDataServlet.java

```diff
@@ -42,6 +42,7 @@
 public class PostDataServlet extends HttpServlet {
   
   public static final DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+  String entityKind = "College";
```

Comment:
Should this be `public static final String ENTITY_KIND = "College"` if it's a constant?

Reply:
Yes, great suggestion